sceptre/scipool/templates/sc-scheduled-jobs-launchrole.yaml

```diff
@@ -0,0 +1,69 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: "ServiceCatalog Scheduled Jobs launch role"
+Resources:
+  SCScheduledJobsLaunchRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      RoleName: SCScheduledJobsLaunchRole
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AmazonLambdaFullAccess
```

Comment:
If the implementation switched from Lambda to Batch, why is this `AmazonLambdaFullAccess`?

Reply:
We need `AmazonLambdaFullAccess` because our lambda triggers the batch job.  Also added `arn:aws:iam::aws:policy/AWSBatchFullAccess`.